chart/templates/configMap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{  .Values.configMapName  }}
```

Comment:
> In general, templates should not define a namespace. This is because Helm installs objects into the namespace provided with the --namespace flag. By omitting this information, it also provides templates with some flexibility for post-render operations (like helm template | kubectl create --namespace foo -f -)

If you are interested in reading more, there is a lot of debate and opinions in this issue https://github.com/helm/helm/issues/5465 The most relavent comments are [here](https://github.com/helm/helm/issues/5465#issuecomment-473942223) and [here](https://github.com/helm/helm/issues/5465#issuecomment-692133971)